src/NuGet.Core/NuGet.PackageManagement/Strings.resx

```diff
@@ -530,4 +530,8 @@
     <value>Package namespace match not found for package ID '{0}'</value>
     <comment>{0} - package id</comment>
   </data>
-</root>
+  <data name="RestoreErrorNU1000" xml:space="preserve">
+    <value>NU1000: Restore error happened, '{0}'.</value>
```

Comment:
I don't think we need extra strings. Report the exception message, just as the old code does, but with the NU1000 error code.